noxfile.py

```diff
@@ -55,28 +85,28 @@ def tests(session: Session) -> None:
 
 
 @session(python=["3.11", "3.12"], name="Django tests", tags=["tests"])
-@nox.parametrize("django", ["4.2.0", "4.1.0", "4.0.0", "3.2.0"])
-def tests_django(session: Session, django: str) -> None:
+@with_gql_core_parametrize("django", ["4.2.0", "4.1.0", "4.0.0", "3.2.0"])
```

Comment:
I think we could skip the matrix for the integration tests, what do you think?

Reply:
There was some tests that failed for 3.3 in integrations mainly in subscriptions. I think we can remove django though

Reply:
ok, let's keep them for now 😊